papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+still results in poorly restored images, especially if extended emission regions are
+present in the image. Because of the PSF convolution an extended emission region
+can decompose into multiple nearby point sources and still lead to good model prediction,
+when compared with the data. Those almost equally good solutions correspond
+to many narrow local minima or "spikes" in the global likelihood surface. Depending
+on the start estimate for the reconstructed image :math:`\mathbf{x}` the RL method will follow
+the steepest gradient and converge towards the nearest narrow local minimum.
+This problem has been described by multiple authors, such as :cite:`Reeves1994`
+and :cite:`Fish95`.
+
+
+The LIRA Multiscale Prior
++++++++++++++++++++++++++
+One possible solution to this problem was described in :cite:`Esch2004`.
+First the standard RL method can be extended by taking into account
+the non uniform exposure :math:`e_i` and a background estimate :math:`b_i`:
+
+.. math::
+   :label: convolution
+
+    \lambda_i = \sum_k (e_i \cdot x_i) p_{i - k} + b_i
+
+Second the authors proposed to extend the Poisson log-likelihood
+function (Eq. :ref:`cash`) by a log-prior term that controls the
+smoothness of the reconstructed
+image on multiple spatial scales. For this the image is transformed
+into a multi-scale representation. Starting from the full resolution
+the image is divided into groups of 2x2 pixels. Each of the groups
+of 2x2 pixels is then divided by their total sum, resulting in
+an image containing the "split proportions" with respect to the
+image down sampled by a factor of two. This process is continued
+to further reduce the resolution of the image until only one
+pixel, containing the total sum of the full-resolution image,
+is left. For each of the 2x2 groups of the re-normalized images
+a Dirichlet distribution is introduced as a prior and summed
+up across all 2x2 groups and resolution levels. For each resolution
+level a parameter :math:`\alpha_k` is introduced, which represents
+the number of "prior counts" added to this resolution level,
+equally to each pixel, which effectively results in a smoothing
+of the image at the given resolution level. The distribution
+of `\alpha` values at each resolution level is described
+by a hyperprior distribution:
+
+.. math::
+   :label: prior
+
+    p(\alpha_k) = \exp{-\delta \alpha^3 / 3}
+
+Resulting in a fully hierarchical Bayesian model. A complete more
+detailed description of the prior definition is given in :cite:`Esch2004`.
+
+
+The Pylira Package
+------------------
+
+Dependencies & Development
+++++++++++++++++++++++++++
+
+The *Pylira* package is a thin Python wrapper around the original *LIRA* implementation provided by
+the authors of :cite:`Connors2011`. The original algorithm was implemented in *C* and made available
+as a package to the *R Language* :cite:`rmath`. Thus the implementation depends on the *RMath* library,
```

Comment:
Original Esch et al algorithm was use for deconvolution/reconstruction, while LIRA gives you the option of the baseline model, so you can look for additional unknown structures and generate image of 'residuals' - this is where you can include the Stein et al method for getting the significance of the additional undetermined features.